examples/landsat-c2-l1/LM01_L1GS_001010_19720908_02_T2/LM01_L1GS_001010_19720908_02_T2.json

```diff
@@ -0,0 +1,301 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "LM01_L1GS_001010_19720908_02_T2",
+  "properties": {
+    "platform": "landsat-1",
+    "instruments": [
+      "mss"
+    ],
+    "created": "2022-03-07T13:41:20.119274Z",
+    "gsd": 60,
+    "description": "Landsat Collection 2 Level-1 Data Product",
+    "eo:cloud_cover": 43.0,
+    "view:off_nadir": 0,
+    "view:sun_elevation": 24.87312023,
+    "view:sun_azimuth": 172.41815593,
+    "proj:epsg": 32625,
+    "proj:shape": [
+      4214,
+      4296
+    ],
+    "proj:transform": [
+      60.0,
+      0.0,
+      358830.0,
+      0.0,
+      -60.0,
+      7953510.0
+    ],
+    "sci:doi": "10.5066/P9AF14YV",
+    "landsat:cloud_cover_land": 46.0,
+    "landsat:wrs_type": "1",
+    "landsat:wrs_path": "001",
+    "landsat:wrs_row": "010",
```

Comment:
Stray observation -- should these be `numeric`? I'm assuming they're provided as strings by USGS?

Reply:
Good question. I see that both the USGS and Microsoft treat them as strings in their STAC Items. I'll be a lemming.